src/Pages/RoomLists/Components/FilterInfo/FilterInfo.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Theme from '../../../../Styles/Theme';
+
+const FilterInfo = () => {
+  const [isClicked, setIsClicked] = useState(false);
+  console.log(isClicked);
+
+  return (
+    <Wrapper>
+      <Category name="test">
+        <SearchResult>검색된 숙소 : 21개</SearchResult>
+        {FILTER_CATEGORY.map((category, index) => {
+          return (
+            <ul>
+              <li
+                key={index}
+                color={isClicked}
+                onClick={() => setIsClicked(!isClicked)}
+              >
+                {category}
+              </li>
+            </ul>
+          );
+        })}
+      </Category>
+    </Wrapper>
+  );
+};
+
+export default FilterInfo;
+
+const FILTER_CATEGORY = [
+  '・추천순',
+  '・평점순',
+  '・낮은 가격순',
+  '・높은 가격순',
+];
+
+const Wrapper = styled.section`
+  height: 7vh;
+  /* border: 3px solid tomato; */
+`;
+
+const Category = styled.ul`
+  display: flex;
+  align-items: baseline;
+  color: ${props => console.log(props)};
+
+  li {
+    color: ${props => props.color && 'red'};
+    color: ${props => console.log(props)};
+    font-size: calc(${Theme.fontSize}*0.9);
+    margin-right: 10px;
+    cursor: pointer;
+  }
+  /* .addColor {
+    color: ${Theme.mainColor};
```

Comment:
```suggestion
    color: ${props => props.theme.mainColor};
```
themeProvider에 theme 객체를 넘겨준 이유는 import 없이도 어디서든 해당 객체를 가져와 쓰기 위함입니다!